README.md

```diff
@@ -17,6 +17,9 @@ To install the library and command line program, use the following:
 
 	go get -u github.com/go-bindata/go-bindata/...
 
+#### For newer versions of go (ex: go1.17+) use
+
+	go install github.com/go-bindata/go-bindata/...@latest
```

Comment:
I'm new to go, but if I install go-bindata with go modules then the syntax is a bit different as well; 

```
go mod init
go get github.com/go-bindata/go-bindata/
```

It may be my inexperience but, if not, then that may merit a note as well.